src/Uno.UI/Generated/3.0.0.0/Windows.UI.Xaml.Controls/CalendarDatePicker.cs

```diff
@@ -518,8 +518,8 @@ public  void SetYearDecadeDisplayDimensions( int columns,  int rows)
 				global::Windows.Foundation.Metadata.ApiInformation.TryRaiseNotImplemented("Windows.UI.Xaml.Controls.CalendarDatePicker", "event EventHandler<object> CalendarDatePicker.Closed");
 			}
 		}
-		#endif
-		#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __MACOS__
+#endif
+#if false || __IOS__ || NET461 || __WASM__ || __MACOS__
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826528)

Reply:
Same as above

Reply:
Same as above.